pennylane/ops/qubit/matrix_ops.py

```diff
@@ -20,7 +20,7 @@
 import numpy as np
 
 import pennylane as qml
-from pennylane.operation import AnyWires, Operation
+from pennylane.operation import AnyWires, Operation, NoDecompositionError
```

Comment:
How come this is being imported here?

Reply:
I think this was Christina, but should be changed now that the custom errors are used

Reply:
```suggestion
from pennylane.operation import AnyWires, Operation
```